mmdet3d/datasets/det3d_dataset.py

```diff
@@ -58,6 +59,8 @@ class Det3DDataset(BaseDataset):
             which can be used in Evaluator. Defaults to True.
         file_client_args (dict, optional): Configuration of file client.
             Defaults to dict(backend='disk').
+        show_ins_statistics (bool, optional): Whether to show the statistics of
```

Comment:
We can set its default to True
show_ins_statistics -> show_statistics

Reply:
Considering we show all statistics of the dataset by default, maybe show_ins_num_var is better if we only want it to be a debugging switch.